templates/generic_pages/general_page.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base_page.html' %}
+
+{% load static %}
+{% load wagtailcore_tags %}
+
+
+
+{% block content %}
+
+    {% include 'includes/generic-intro.html' %}
+
+    <div class="container pt-4">
+        {% for block in page.body %}
+            <div class="row">
```

Comment:
Should this `<div>` be here? Maybe it should be on the outside of the `{% for %}` loop?

Reply:
So every Paragraph block has its own div tag.

Reply:
Okay, but was this a requirement? It's a little unusual to wrap individual blocks in extra HTML, as each block type has its own template to control the output.

Reply:
Yes I can see this :                                                                                                                                                           <div class="row">               
<div class="content-block">
    <div class="container">
        <p data-block-key="38tqm">General here</p>
    </div>
  </div>
</div>

Reply:
I will remove it , just to mention we may have the same issue on the home page